package.json

```diff
@@ -153,33 +185,149 @@
                 "description": "Only used for SonarCloud",
                 "default": ""
               }
+            },
+            "additionalProperties": false
+          }
+        },
+        "sonarlint.connectedMode.connections.sonarqube": {
+          "type": "array",
+          "scope": "application",
+          "default": [],
+          "markdownDescription": "Configure connection(s) to [SonarQube](https://sonarqube.org). Don't forget to also configure the project binding in `#sonarlint.connectedMode.project#`.\n\nExample:\n\n    \"sonarlint.connectedMode.connections.sonarqube\": [\n      {\n        \"serverUrl\": \"https://sonar.mycompany.com\",\n        \"token\": \"V2VkIE1...\"\n      }\n    ]\n\nSpecify a `connectionId` if you want to define multiple connections.",
+          "examples": [
+            {
+              "serverUrl": "https://<myServerUrl>",
+              "token": "<mySQToken>"
+            },
+            {
+              "connectionId": "<myConnectionId>",
+              "serverUrl": "https://<myServerUrl>",
+              "token": "<mySQToken>"
+            }
+          ],
+          "items": {
+            "type": "object",
+            "properties": {
+              "connectionId": {
+                "type": "string",
+                "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarQube/SonarCloud.",
```

Comment:
```suggestion
                "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarQube.",
```

Reply:
Here I was considering the case of people having a mix of SQ/SC connections. The id should be unique across all kind of connections.